desktop/core/src/desktop/js/apps/tableBrowser/metastoreTable.js

```diff
@@ -320,7 +320,9 @@ class MetastoreTable {
           const found =
             analysis.properties &&
             analysis.properties.some(property => {
-              if (property.col_name.toLowerCase() === 'view original text:') {
+              if (
+                ['view original text:', 'original query:'].includes(property.col_name.toLowerCase())
```

Comment:
As this will be run for each property how about using a Set instead? I.e. before line 320, `const viewColumnNames = new Set(['view ...']);`?

Reply:
Done.

Also check for `analysis.is_view` but still not confident that it's always true for all views (found in one test scenario for the value as false for one view)

Plus looking at the code, we anyway have to loop around the properties to find the column and view query, so I've kept the check as it is to only look for column names.


On a side note, historically I think both Hive and Impala had same column, but then in newer versions they are different now and this issue is coming up I think. (Tried checking in CDH6 versions).